### PR TITLE
Color of buttons inside dialogs is not applied if there'are nested css styles

### PR DESCRIPTION
**Pull request by zh-m**
The color of buttons inside dialogs is not modified in case the following CSS is defined:

Page Button {
    background-color: #30BCFF;
    color: #FFFFFF;
}

 Button {
    font-size: 15;
    color: springgreen; 
}

In this case, the color of a button inside dialog will be still "#ffffff", while the desired one is "springgreen".

### CODE REVIEW

File: tns-core-modules/ui/dialogs/dialogs.android.ts [hunk 1]
```diff
@@ -39,6 +39,20 @@ function showDialog(builder: android.app.AlertDialog.Builder) {
             }
         }
     }
+
+    var backgroundColor = dialogsCommon.getButtonColor();
```
Comment (hshristov) [inline tns-core-modules/ui/dialogs/dialogs.android.ts]: This looks like foregroundColor not background.

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by enchev** [comment]
run ci

**Pull request merged**